### ISSUE TITLE
Import error

### ISSUE THREAD

**Issue by JackBeauchampStansfield-Which** [anaynayak/aws-cloudwatch-annotations#134]
For the last couple of days we've been getting the below import error:

```
file:///Documents/node_modules/aws-cloudwatch-annotations/lib/cloudwatch.js:1
import { CloudWatchClient, GetDashboardCommand, PutDashboardCommand } from "@aws-sdk/client-cloudwatch";
         ^^^^^^^^^^^^^^^^
SyntaxError: Named export 'CloudWatchClient' not found. The requested module '@aws-sdk/client-cloudwatch' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from '@aws-sdk/client-cloudwatch';
const { CloudWatchClient, GetDashboardCommand, PutDashboardCommand } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:105:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:151:5)
    at async Loader.import (node:internal/modules/esm/loader:166:24)
    at async Object.loadESM (node:internal/process/esm_loader:68:5)
```

**Comment by anaynayak** [comment]
Thanks for reporting this @jbs-which . Will investigate this further and get it fixed.

**Comment by JackBeauchampStansfield-Which** [comment]
Awesome, thank you. The suggested fix seemed to work when I did some very basic troubleshooting locally but node is very much not my strong suit so will leave it to you!

**Comment by anaynayak** [comment]
I believe this should now be fixed with v1.2.2

`npx aws-cloudwatch-annotations@1.2.2`
